tests/ops/api/v1/endpoints/test_user_endpoints.py

```diff
@@ -1434,12 +1434,6 @@ def test_update_system_manager_existing_system_not_in_request_which_removes_syst
                 "description": "fixture-made-system",
                 "organization_fides_key": "default_organization",
                 "system_type": "Service",
-                "data_responsibility_title": "Processor",
```

Comment:
We could also remove this whole test file: `tests/ops/migration_tests/test_system_dictionary_data_migration.py`

Reply:
ah i did consider removing that but i felt that the migration itself will still technically run, so in some senses, it's still valid to have those tests in place?? anyway, didn't see a whole lot of harm in keeping it in place because it was all passing without needing any updates 🤷